jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/util/JKubeProjectUtil.java

```diff
@@ -169,4 +169,12 @@ public static URLClassLoader getClassLoader(JavaProject jKubeProject) {
         jKubeProject.getCompileClassPathElements(),
         jKubeProject.getOutputDirectory().getAbsolutePath());
   }
+
+  public static Optional<List<String>> getCompileClasspathElementsIfRequested(JavaProject project, boolean useProjectClasspath) {
+    if (!useProjectClasspath) {
+      return Optional.empty();
+    }
+
+    return Optional.of(project.getCompileClassPathElements());
+  }
```

Comment:
As agreed during meeting, let's inline this where applicable (2 Mojos).

`useProjectClasspath ? project.getCompileClassPathElements() : Collections.emptyList();`

Once Mojos extend AbstractJKubeMojo, this should be moved to a method.

For KubernetesExtension, we can go directly for the method and use it where applicable.